ddtrace/tracer/rules_sampler.go

```diff
@@ -70,6 +70,15 @@ type SamplingRule struct {
 	// If not specified, the default is no limit.
 	MaxPerSecond float64
 
+	// Resource specifies the regex pattern that a span resource must match.
+	Resource *regexp.Regexp
+
+	// Tags specifies the map of key-value patterns that span tags must match.
+	Tags map[string]*regexp.Regexp
+
+	// TargetSpan specifies whether sampling should be applied to the root spans only.
+	TargetSpan bool
```

Comment:
If we want to keep the boolean here we should either name it `TargetRoot` or `TargetRootSpan`. If we want to keep `TargetSpan` it should be a string that can have `root`, `any` or `""` values.